### PR TITLE
Mh/hunter dog/class feats

### PR DESCRIPTION
**Pull request by bleehu**
Removed old nanite feats that were out of aesthetic and replaced them with more hunter-y class feats. Attempting to fulfill the design requirements laid out here: https://github.com/bleehu/CX_Design_Guide/blob/master/characters/how_to_classes.md#hunter

### CODE REVIEW

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -448,32 +449,60 @@ Dog Personalities:
...
+tracked this encounter, you may do so as a Free Action instead of an Action.
+	For example, let's say a Hunter was fighting an enemy robot and an enemy 
+soldier. The first turn, the Hunter spends one Action to track the robot. The
+second turn, the Hunter spends an Action to track the soldier. The third turn 
+the Hunter tracks the robot again, but this time it is a Free Action, so the 
+Hunter still has 4 Actions left on his turn. Continuing his third turn, the 
+Hunter also tracks soldier. Even though the Hunter already tracked the soldier 
+before, the hunter has used up his one Free Tracking Action this turn, and 
+tracking the soldier again costs one Action.
+	In addition, teammates now get a bonus to crit on tracked enemies that they
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: terminology: "crit"

File: CharacterCreation/02_Classes.txt [hunk 2]
```diff
@@ -448,32 +449,60 @@ Dog Personalities:
...
+hit in accordance to their range and weapon type. This is accomplished by you
+calling out weak spots that you see on the target and describing shooting 
+conditions such as the bullet drop due to humidity, wind speed and direction, 
+and the coriollis effect making the bullet swerve. The bonus to crit is listed
+below. Note that if the chance to crit exceeds the chance to hit, it does not
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "crit"

File: CharacterCreation/02_Classes.txt [hunk 3]
```diff
@@ -448,32 +449,60 @@ Dog Personalities:
...
+the Hunter tracks the robot again, but this time it is a Free Action, so the 
+Hunter still has 4 Actions left on his turn. Continuing his third turn, the 
+Hunter also tracks soldier. Even though the Hunter already tracked the soldier 
+before, the hunter has used up his one Free Tracking Action this turn, and 
+tracking the soldier again costs one Action.
+	In addition, teammates now get a bonus to crit on tracked enemies that they
+hit in accordance to their range and weapon type. This is accomplished by you
+calling out weak spots that you see on the target and describing shooting 
+conditions such as the bullet drop due to humidity, wind speed and direction, 
+and the coriollis effect making the bullet swerve. The bonus to crit is listed
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "crit"

File: CharacterCreation/02_Classes.txt [hunk 4]
```diff
@@ -448,32 +449,60 @@ Dog Personalities:
...
+below. Note that if the chance to crit exceeds the chance to hit, it does not
+increase the chance to hit; it only garuntees that a hit will be a critical hit.
+
++-------+------------+-----------------------------+
+| Range | Crit bonus | Crit bonus with Long Rifles |
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "crit" "crit"

**Review by rosebrallier — APPROVED** [review]
Marvelous work! I marked all the instances to "crit" for reference.

**Comment by bleehu** [comment]
Whoops! Will fix all!

**Pull request merged**